src/app/src/components/FacilityListItemsTable.jsx

```diff
@@ -192,29 +189,29 @@ function FacilityListItemsTable({
         });
 
     return (
-        <Paper style={facilityListItemsTableStyles.containerStyles}>
-            <div style={facilityListItemsTableStyles.tableWrapperStyles}>
-                <Table>
-                    <TableHead>
-                        {paginationControlsRow}
-                        <FacilityListItemsTableRow
-                            rowIndex="CSV Row Index"
-                            countryName="Country Name"
-                            name="Name"
-                            address="Address"
-                            status="Status"
-                            hover={false}
-                        />
-                    </TableHead>
-                    <TableBody>
-                        {tableRows}
-                    </TableBody>
-                    <TableFooter>
-                        {paginationControlsRow}
-                    </TableFooter>
-                </Table>
-            </div>
-        </Paper>
+        <>
```

Comment:
Is this empty tag intentional/required?

Reply:
unintentional

Reply:
This shouldnt be required. I'll remove it.